policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax/mi_total_homestead_property_tax_credit.py

```diff
@@ -0,0 +1,74 @@
+from policyengine_us.model_api import *
+
+
+class mi_total_homestead_property_tax_credit(Variable):
```

Comment:
total suggests final
```suggestion
class mi_allowable_homestead_property_tax_credit(Variable):
```